docs/src/pages/index.js

```
@@ -230,12 +310,59 @@ function HomepageHeader() {
           <div
             className={styles.npmInstallBox}
-            onClick={handleCopyNpm}
+            onClick={handleCopy}
+            onKeyDown={handleCopyKeyDown}
+            role="button"
+            tabIndex={0}
             title={copied ? 'Copied!' : 'Click to copy'}
           >
-            <code>pnpm add @allxsmith/bestax-bulma</code>
+            <code>{command}</code>
             {copied ? <CheckIcon /> : <CopyIcon />}
```

ESLint

[error] Error: Cannot create components during render

Components created during render will reset their state each time they are created. Declare components outside of render.

  318 |           >
  319 |             <code>{command}</code>
> 320 |             {copied ? <CheckIcon /> : <CopyIcon />}
      |                        ^^^^^^^^^ This component is created during render
  321 |           </div>
  322 |           {/*
  323 |             Outside the copy box on purpose: nesting these inside it would bubble

  280 |
  281 |   // Check Icon SVG (for copied state)
> 282 |   const CheckIcon = () => (
      |                     ^^^^^^^
> 283 |     <svg
      | ^^^^^^^^
> 284 |       width="16"
      …
      | ^^^^^^^^
> 291 |     </svg>
      | ^^^^^^^^
> 292 |   );
      | ^^^^ The component is created during render here
  293 |
  294 |   return (
  295 |     <header className={clsx('hero hero--primary', styles.heroBanner)}>

(react-hooks/static-components)